commit 6ea926065eaf4914203daa09a5945c9aefb06be5 (refs/changes/69/5569/3)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-02-15T14:59:09+00:00 (8 months ago)
    
    TRITON-1220 node-tap versions between 12.0.1 and 12.5.2 can emit invalid TAP
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/package.json b/package.json
index 4d84ff9..e8ee531 100644
--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "eslint": "4.13.1",
     "eslint-plugin-joyent": "~2.0.0",
     "mock-fs": "4.5.x",
-    "tap": "^12.0.1",
+    "tap": "^12.5.2",
     "zkstream": "0.11.2"
   },
   "engines": {
